### ISSUE TITLE
Add Right-click fine slider pop-up for all "layer controls" sliders

### ISSUE THREAD

**Issue by pamitabh** [napari/napari#7297]
## 🚀 Feature
Right now only the "contrast limits:" shows a pop-up slider when right-clicked, proposing this feature to be added consistently to all the sliders shown in layer controls, irrespective of the layer.

## Motivation and Pitch
I have found myself trying to adjust other properties using the sliders, like image layer: gamma, opacity; track layer: tail length, width, etc. points: opacity, point size, etc. It is hard to do it accurately using only the present control bar. It will be more accurate to use the **big "fine control" slider** with an option to enter the range directly. 

This will also make the UI/UX more consistent across all sliders shown in layer control, irrespective of the layer type.

## Alternatives
None

**Comment by jni** [comment]
Yeah, we have discussed this a few times at community meetings and people are in favour! We just haven't had time to implement.

Potentially, this would be a good contribution at [superqt](https://github.com/pyapp-kit/superqt), and then we would just use it.

But we'd also consider a pull request implementing things here, which we could later upstream to superqt.

If you have the time and inclination to work on this, we would be happy to spend some time pairing with you to help out!

**Comment by pamitabh** [comment]
> Yeah, we have discussed this a few times at community meetings and people are in favour! We just haven't had time to implement.

It’s great to hear that others feel the same way!

> If you have the time and inclination to work on this, we would be happy to spend some time pairing with you to help out!

Sadly, I don't have any experience with gui or qt development.
